### DISCUSSION TITLE
Microsoft Account will not signe in my GitHub account?

### DISCUSSION THREAD

**Discussion by Samuelfcampbell2003** [community/community#44361] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Microsoft Account log-in options automatically offers the button to log-in  with my GitHub account? 

I clicked that button. 
I was redirected to the log-in Github page. 
Put in my email and password.
A a pop up offered to save info for browser, and Notified me an email with a code was sent to both my primary and secondary email accounts. 
Once I put the code into the windows, I was notified this site was not... and I could try again later. 

I have no idea if this is a Personal issue, Microsoft or a Github issue. The more I think about it. I've been having other issues loging-in to my Microsoft account. Thank you all have a great weekend. 
~ Samuel F Campbell

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Samuelfcampbell2003 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Samuelfcampbell2003** [reply]
I appreciate your understanding, and Thank You!
